app/models/user.rb

```diff
@@ -1,23 +1,26 @@
 class User < ApplicationRecord
+  scope :except_admins, -> { where(role: [:user, :manager]) }
```

Comment:
lets go with something alongs these lines....... where.not(role: :admin)
Its more scalable if new roles are added.

It is fine for our case, but just something you could use in your future thought process.